torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2280,6 +2280,32 @@ def sample_inputs_elementwise_binary(op, device, dtype, requires_grad, **kwargs)
 
         yield SampleInput(lhs, args=(rhs,), kwargs=sample_kwargs, broadcasts_input=broadcasts_input)
 
+def sample_inputs_jiterator(op, device, dtype, requires_grad, **kwargs):
+    shapes = (
+        ((), ()),
+        ((S,), ()),
+        ((S, 1), (S,)),
+        ((M, S), ()),
+        ((S, M, S), (M, S)),
+        ((S, M, S), (S, M, S)),
+        ((M, 1, S), (M, S)),
+        ((M, 1, S), (1, M, S)),
+        ((0, 1, 3), (0, 10, 3))
+    )
+
+    num_inputs = kwargs.get('num_inputs')
+    sample_kwargs = kwargs.get('sample_kwargs', {})
+
+    for shape_lhs, shape_rhs in shapes:
+        lhs = make_tensor(shape_lhs, device=device, dtype=dtype, requires_grad=requires_grad)
```

Comment:
fyi, a common pattern which might save you some typing in the future is to partial `make_tensor` to always insert the device, dtype, and requires_grad info

Reply:
Thanks!